src/components/Note/NoteReactionList/NoteReactionChipCondensed/NoteReactionChipCondensed.tsx

```diff
@@ -5,6 +5,8 @@ import {uniqueId} from "underscore";
 import {TooltipPortal} from "components/TooltipPortal/TooltipPortal";
 import {ReactionModeled} from "../NoteReactionList";
 import "./NoteReactionChipCondensed.scss";
+import {useAppSelector} from "../../../../store";
+import {getEmojiWithSkinTone} from "../../../../utils/reactions";
```

Comment:
one last nitpick 😄 (don't forget to put the `scss` as last import)

Reply:
😄

Reply:
the stylesheet still isn't the last import 😭